src/ConfigProvider.php

```diff
@@ -21,7 +23,7 @@ public function __invoke()
      *
      * @return array
      */
-    public function getPaginatorConfig()
+    public function getPaginatorConfig(): array
```

Comment:
Adding type declarations here is a problem, because this class is not `final`, so subtypes are potentially broken.

Reply:
Removed in new PR